fab/fabfile.py

```diff
@@ -332,8 +332,8 @@ def hotfix_deploy():
     run('echo ping!')  # workaround for delayed console response
     try:
         execute(release.update_code, env.deploy_metadata.deploy_ref, True)
-    except Exception:
-        execute(mail_admins, "Deploy failed", "You had better check the logs.")
+    except Exception, e:
```

Comment:
nit picky, but I think `except Exception as e` is preferred to avoid the confusion when trying to catch errors of multiple types